src/endpoint/Timesheet.Delete/FlowStep/DateWebAppCacheJson.cs

```diff
@@ -0,0 +1,4 @@
+﻿namespace Flow.FlowStep
```

Comment:
Rename namespace to `GarageGroup.Internal.Timesheet`. Use this namespace within whole project. Also use scope namespace:

```
namespace GarageGroup.Internal.Timesheet;

internal sealed record DateWebAppCacheJson(string? ActivityId);
```